fastlane/lib/fastlane/actions/increment_build_number.rb

```diff
@@ -29,7 +29,7 @@ def self.run(params)
         # More information about how to set up your project and how it works:
         # https://developer.apple.com/library/ios/qa/qa1827/_index.html
         # Attention: This is NOT the version number - but the build number
-        agv_enabled = system([command_prefix, 'agvtool what-version', command_suffix].join(' '))
+        agv_enabled = system([command_prefix, 'agvtool what-version', command_suffix].join(' ')) unless Helper.test?
```

Comment:
This is confusing to me, but maybe I'm crazy. `agv_enabled = (something) unless we're in test mode.`
would it make sense to more explicitly say it is disabled instead.